### ISSUE TITLE
 Error: Accessibility Violations, Rule: skip-link

### ISSUE THREAD

**Issue by oene** [open-wc/open-wc#1378]
Hi,

I'm a newbe and trying to run the example project. 

Running npm run test I get the following error:

HeadlessChrome 79.0.3945 (Windows 10.0.0) DemoTag passes the a11y audit FAILED
        Error: Accessibility Violations
        ---
        Rule: skip-link
        Impact: moderate
        The skip-link target should exist and be focusable (https://dequeuniversity.com/rules/axe/3.5/skip-link?application=axeAPI)

        Issue target: demo-tag,.active
        Context: <a href="#main" class="active">
                      Main
                    </a>
        Fix any of the following:
          No skip link target

        Issue target: demo-tag,a[href$="\#pageOne"]
        Context: <a href="#pageOne" class="">
                      Page One
                    </a>
        Fix any of the following:
          No skip link target

        Issue target: demo-tag,a[href$="\#about"]
        Context: <a href="#about" class="">
                      About
                    </a>
        Fix any of the following:
          No skip link target
        ---
Am I doing something wrong? Google didn't provide a answer for this.

Thnx!

Oene

**Comment by thepassle** [comment]
Hi,

I'm sorry you're running into this! I have a few questions:

Did you scaffold the project with `npm init @open-wc`? I just scaffolded a new project and ran the tests, and everything seems to pass for me. Would it be possible for you to provide a github repo with a reproduction?

```bash
⚡ npm test

> meetup-demo@0.0.0 test /Users/au87xu/owc-meetup-demo/meetup-demo
> karma start --coverage


START:
27 02 2020 17:07:59.223:INFO [karma-server]: Karma v4.4.1 server started at http://0.0.0.0:9876/
27 02 2020 17:07:59.228:INFO [launcher]: Launching browsers ChromeHeadlessNoSandbox with concurrency unlimited
27 02 2020 17:07:59.233:INFO [launcher]: Starting browser ChromeHeadless
27 02 2020 17:07:59.662:INFO [HeadlessChrome 80.0.3987 (Mac OS X 10.14.5)]: Connected on socket 5SGrNvTVnMky_KC2AAAA with id 89529893
  MeetupDemo
    ✔ has page "main" by default
    ✔ renders default fallback content
    ✔ renders page-one if page property is set to pageOne
    ✔ changes the page if a menu link gets clicked
    ✔ matches the snapshot
    ✔ passes the a11y audit
  PageMain
    ✔ has a default title "Hello open-wc world!"
    ✔ can override the title via attribute
    ✔ matches the snapshot
    ✔ passes the a11y audit
  PageOne
    ✔ has a default title "Hey there" and counter 5
    ✔ increases the counter on button click
    ✔ can override the title via attribute
    ✔ shows initially the text "hey there Nr. 5!" and an "increment" button
    ✔ passes the a11y audit

Finished in 0.245 secs / 0.128 secs @ 17:08:00 GMT+0100 (Central European Standard Time)

SUMMARY:
✔ 15 tests completed
TOTAL: 15 SUCCESS

=============================== Coverage summary ===============================
Statements   : 100% ( 31/31 )
Branches     : 100% ( 4/4 )
Functions    : 100% ( 16/16 )
Lines        : 100% ( 31/31 )
================================================================================
```

**Comment by oene** [comment]
Hi

Put the project on Git
https://github.com/oene/owc-demo-tag

Thnx for the help!

**Comment by oene** [comment]
If I select Application the above error appears.
If I select Web component then there are no errors.

**Comment by LarsDenBakker** [comment]
We were able to reproduce this one, it's a quirk in axe. We're deciding how to deal with it properly.

**Comment by thepassle** [comment]
I've opened an issue at axe-core here: https://github.com/dequelabs/axe-core/issues/2069

**Comment by thepassle** [comment]
Looks like its a known issue, tracked here: https://github.com/dequelabs/axe-core/issues/2055

**Comment by thepassle** [comment]
Update: https://github.com/dequelabs/axe-core/issues/2069#issuecomment-595861481

**Comment by LarsDenBakker** [comment]
Closing this since the upstream PR was merged.
